src/main/java/net/runelite/rsb/wrappers/subwrap/NPCChat.java

```diff
@@ -17,10 +21,32 @@ public String getName() {
     }
 
     public String[] getOptions() {
-        return new String[]{};
+        try {
+            return Arrays.stream(getComponent(GlobalWidgetInfo.DIALOG_DYNAMIC_CONTAINER).getComponents()).map(RSWidget::getText).toArray(String[]::new);
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    public boolean containsOption(String option){
```

Comment:
fix spaces pls

```java
containsOption(String option) {
```

Reply:
refactored a bit and appended some generic comments.